website/static/index.html

```diff
@@ -3,13 +3,13 @@
 <html data-wf-page="65402b66d39d6454e51fabf1" data-wf-site="65402b66d39d6454e51fabed">
 <head>
   <meta charset="utf-8">
-  <title>OneTable</title>
-  <meta content="OneTable is an omni-directional interop of lakehouse table formats Apache Hudi, Apache Iceberg, and Delta Lake. OneTable is NOT a new or separate format, OneTable provides abstractions and tools for the translation of lakehouse table format metadata." name="description">
-  <meta content="OneTable" property="og:title">
-  <meta content="OneTable is an omni-directional interop of lakehouse table formats Apache Hudi, Apache Iceberg, and Delta Lake. OneTable is NOT a new or separate format, OneTable provides abstractions and tools for the translation of lakehouse table format metadata." property="og:description">
+  <title>XTable</title>
```

Comment:
The title should be "Apache XTable™" as well as the other prominent references to XTable in this page.